src/integrationTest/java/uk/gov/hmcts/darts/utils/MtomDemoClient.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.darts.utils;
+
+import org.junit.jupiter.api.Assertions;
+import uk.gov.hmcts.darts.workflow.command.AddAudioMidTierCommand;
+import uk.gov.hmcts.darts.workflow.command.Command;
+import uk.gov.hmcts.darts.workflow.command.CommandFactory;
+
+import java.io.IOException;
+
+public final class MtomDemoClient {
```

Comment:
Struggling to see how the original ticket for this change has resulted in a MTOM Demo Client app?

Reply:
I have removed these. This was purely convenience and creates noise